first-day/building/index.html

```diff
@@ -0,0 +1,182 @@
+     <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/languages/java.min.js"></script>
+     <script>hljs.initHighlightingOnLoad();</script>
+
+     
+
+     <style>
+       
+
+body 
+{
+  font-family: "Open Sans", "Helvetica", "Helvetica Neue",  "Arial", sans-serif;
+}
+ 
+.content
+{
+  padding:10px;
+}
+
+figcaption
+{
+  margin-bottom: 20px;
+}
+
+.vertical-align
+{
+  display: flex;
+  align-items: center;
+}
+
+.ui.segment.pushable
+{
+  margin: 0;
+  padding: 1rem 0;
+  overflow: visible;
+}
+
+h1
+{
+  font-style:italic;
+  font-size:130%;
+}
+h2
+{
+  font-size:110%;
+  border-bottom: thin solid black;
+}
+h3
+{
+  font-size:100%;
+  border-bottom: thin solid black;
+}
+
+
+
+
+     </style>
+
+   </head>
+
+  <body>
+    
+
+  
+  <header class="ui center aligned middle aligned stackable grid segment">
+
+    <div class="ui five wide column">
+      <h2> Building Rules and Info </h2>
+    </div>
+
+    <div class="ui two wide column">
+      
+          
+  <div class="two wide column sitemap">
+    <div class="popup" data-content="To the top level Topics for this Module...">
+      <a href="../index.html"> <i class="huge home icon"></i> <br> Module Home </a>
+    </div>
+  </div>
+
+        
+    </div>
+
+    <div class="ui five wide column">
+      
+  <div class="four wide column">
+    <div class="ui center aligned mini message segment">
+      Redhat Mobile, Waterford.
+      <a href="http://creativecommons.org/licenses/by-nc/4.0/"
+         title="External link to http://creativecommons.org/licenses/by-nc/4.0/"
+         target="_blank">Creative Commons License
+      </a>
+    </div>
+  </div>
+
+    </div>
+
+    <div class="three wide column">
+      
+    </div>
+
+
+  </header>
+    <div class="content">
+ 
+      <h1>General Rules</h1>
+
+      <h2>Rules: In general common sense should prevail!</h2> 
+      <ol>
+        <li>This is a place of work and we have our own health, fire and safety rules which we will share and require all interns to follow</li>
+
+        <li>We do not have a formal dress code, casual clothes is best but tracksuits would not be appropriate. Hoodies are fine once they are tasteful</li>
+
+        <li>We have a canteen area and most people bring in their own lunch. We discourage leaving the office.</li>
+
+        <li>We clean up after ourselves, cups / plates etc in the dishwasher, rubbish in the bin</li>
+
+        <li>We have free tea/coffee/snacks program which everyone is respectful of and we do not overly abuse it. It's not a substitute for lunch.</li> 
+
+        <li>The rules that you adhere to in school extend to the workplace e.g. profanity, noise level, have respect for peers and colleagues. </li>
+
+        <li>We are customer facing at times and a zero tolerance approach will be taken for any incidents that bring your school or Red Hat into disrepute</li>
```

Comment:
Would look to add the following:

- note about hardware
- note about the use of internet
- note about noise/ use of meeting rooms/ availability of whiteboards

Reply:
+1

The hardware will be shared, don't grow attached to it. If you have a laptop yourself feel free to bring it in

Internet is open by default but whatever you do will be logged. We won't tolerate any illegal activity on our network and questionable sites are not.

Third point Damien makes is in reference to the fact this is a workplace, respect others and use the meeting rooms and keep noise to a minimum

Reply:
@lgriffin @damienomurchu updated in next commit.